pkgs/applications/misc/dstask/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "dstask";
+  version = "0.17";
+
+  src = fetchurl {
+    url = "https://github.com/naggie/dstask/releases/download/${version}/dstask-linux-amd64";
```

Comment:
Please make the platform/arch a parameter of the url and/or restrict the platform accordingly with a proper attribute in the meta set.

Reply:
Thank you for the feedback. Is this latest commit doing it correctly? https://github.com/NixOS/nixpkgs/pull/87383/commits/586ae5a10cd8b79f2c406a3e36c5ff2cacd91ffe